homeassistant/components/weather/__init__.py

```diff
@@ -221,6 +222,28 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         ],
         supports_response=SupportsResponse.ONLY,
     )
+    component.async_register_entity_service(
+        SERVICE_GET_FORECAST_VALUE,
+        {
+            vol.Required("attribute"): vol.In(
+                (
+                    "temperature",
+                    "templow",
+                    "precipitation",
+                    "precipitation_probability",
+                    "wind_speed",
+                    "humidty",
+                )
+            ),
+            vol.Optional("limit_hours"): vol.Range(1, 216),
```

Comment:
```suggestion
            vol.Optional("limit_hours"): vol.Range(1, 24 * 9),
```

Reply:
Why not 10 days ?
```suggestion
            vol.Optional("limit_hours"): vol.Range(1, 24 * 10),
```

Reply:
I got only 216 forecasted hours as output. I am implementing a check to avoid array out of range exceptions, so it doesn't matter

Reply:
I think it depends on weather integrations and/or location to provide different forecasted hours range.

So out of range should be handled nicely to prevent errors.

And I think we prefer a calculated value to easily see how long the limit is 9/10 days (216 = 24 * 9)